tools/build-makefile

```diff
@@ -0,0 +1,224 @@
+#  !---------------------------------------------------------------------!
```

Comment:
It seems dangerous to have build definitions separated from the source tree.

Reply:
I also don't fully understand this but it seems to duplicate at least part of the information in `src/libxc/Makefile` and `src/cuda/Makefile`.